installing/upgrade/upgrade-to-r3.0.md

```diff
@@ -100,7 +100,7 @@ Now, when you have dom0 upgraded, you can install new templates from Qubes R3.0
 Upgrading template on already upgraded dom0
 -------------------------------------------
 
-When for some reason you did not upgraded all the templates and standalone VMs before upgrading dom0, you can still do this, but it will be somehow more complicated. This can be a case when you restore backup done on Qubes R2.
+If for some reason you did not upgrade all the templates and standalone VMs before upgrading dom0, you can still do this, but it will be somewhat more complicated. This can be the case when you restore backup done on Qubes R2.
```

Comment:
somehow still sounds wrong to me, it would be saying 'we know it will be more complicated but we don't know why this is'

'somehow' only makes sense if you are trying to imply there's a mystery about it ('somehow it is better than it was before') or that you don't know how to do something, only that it must be done e.g 'we must achieve this somehow'